specification/advisor/resource-manager/Microsoft.Advisor/preview/2016-07-12-preview/advisor.json

```diff
@@ -398,6 +398,72 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Advisor/advisorScore": {
+      "get": {
+        "tags": [
+          "AdvisorScore"
+        ],
+        "description": "Retrieves data about the advisor score of the subscription",
+        "operationId": "AdvisorScore_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/advisorScore"
+              }
+            }
+          }
+        },
+        "deprecated": false,
+        "x-ms-examples": {
+          "ListAdvisorScore": {
+            "$ref": "./examples/ListAdvisorScore.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Advisor/advisorScore/{name}": {
+      "get": {
+        "tags": [
+          "AdvisorScore"
+        ],
+        "description": "Retrieves data about the advisor score of the subscription",
+        "operationId": "AdvisorScore_Get",
+        "parameters": [
```

Comment:
Could we please order this list as the same order in the URL? In this case, subscription ID parameter goes first and then the name.
This would affect some generated SDK